src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -624,7 +618,8 @@ public static String sanitizeName(String type, String name) {
 	}
     // Windows Explorer don't like below special characters in Folder names so zip extraction fails. Replacing with _
 	public static String sanitizeFolderNames(String zipFileName) {
-		return zipFileName.replaceAll("[?>|<:]", "_");
+		zipFileName=zipFileName.replace('\\','_');
```

Comment:
added comments with regards to  `\\` and replace all of them with underscore Like the `folder \ this \ name` with the `folder_this_name`